### ISSUE TITLE
et al formatting in group citation

### ISSUE THREAD

**Issue by vanukalla** [JabRef/jabref#3723]
I added some stylefiles in the jstyle section on github (as issue) and am working on another one. For this one I need author-year citations with the et al in italics - like this:

(Nader _et al_, 1983; Rueffer _et al_, 2001; Di Bisceglie _et al_, 2013)

What I do get is:
(Nader _et al_, 1983; Rueffer et al, 2001; Di Bisceglie et al, 2013)  

The first et al is in italics, the rest not. If the citations are added separately, they are put separately and the et al string is italic in both:

(Yang _et al_, 2011) (Campo _et al_, 2011) 

In the stylefile, the citation arguments are defined like this:
CITATION
AuthorField=author/editor
YearField="year"
MaxAuthors=2
AuthorSeparator=", "
AuthorLastSeparator=" & "
EtAlString=" et al"
ItalicEtAl="true"
CitationSeparator="; "
MultiCiteChronological=true
BracketBefore="("
BracketAfter=")"
CitationCharacterFormat="Default Style"
FormatCitations="true"
UniquefierSeparator=", "
InTextYearSeparator=" "
YearSeparator=", "

So according to documentaion, the et al should be in italics. Why is, in a group, only the first et al formatted and not the rest?

**Comment by lenhard** [comment]
Thanks for reporting!

This is not really down to JabRef, as far as I know. We just use [citeproc-java](https://michel-kraemer.github.io/citeproc-java/) for processing the style files and formatting the output. We don't do any customization here. So, you should probably report the issue there.

**Comment by Siedlerchr** [comment]
@lenhard  In this case it is part of JabRef. JStyle files are the files JabRef uses for formatting citations in OpenOffice/LibreOffice. That's our custom things. 
In the future we should get rid of our custom thing and use CSL #2146

**Comment by lenhard** [comment]
Darn... Sorry for the confusion and thanks for correcting me @Siedlerchr

**Comment by subhramit** [comment]
Is this issue solved now?

**Comment by koppor** [comment]
@subhramit I don't think so. Could be a good issue to dive into the OO/LO code of JabRef.

**Comment by subhramit** [comment]
Could someone try reproducing this?
I tried, and for some reason it works alright in mine:

![image](https://github.com/JabRef/jabref/assets/74734844/7b0f6eba-c7fc-4277-b0cc-3ee435aa4c35)

(I used the same stylefile as in the PR description, with ```CitationCharacterFormat="Default Style"``` removed as it was giving some trouble, could that have affected anything?)

**Comment by vanukalla** [comment]
I`ll be happy to test this but earliest on Tuesday.

**Comment by vanukalla** [comment]
You are absolutely right - it is working for me, too! And I used my original style file which gave the issue at the time when I observed this issue.

![citation](https://github.com/JabRef/jabref/assets/32541553/48d880ab-ac86-456e-ba16-5508e85b9b6b)

Might as well have been some problem with the LO version at that time.
I consider this issue as solved.
Thanks for testing, subhramit!

**Comment by subhramit** [comment]
> You are absolutely right - it is working for me, too! And I used my original style file which gave the issue at the time when I observed this issue.
> 
> ![citation](https://github.com/JabRef/jabref/assets/32541553/48d880ab-ac86-456e-ba16-5508e85b9b6b)
> 
> Might as well have been some problem with the LO version at that time.
> I consider this issue as solved.
> Thanks for testing, subhramit!

No issues, thank you for checking! :)
